pages/index.js

```diff
@@ -49,32 +86,73 @@ export default function ShoppingItemsList({
             mode={mode}
           />
         )}
-
-        {mode !== "add" && (
-          <AddItemContainer>
-            <>
-              <p>...need anything else?</p>
+        {mode !== "add" ||
+          (!mode && (
+            <AddItemContainer>
+              <>
+                <p>...need anything else?</p>
+              </>
               <AddButton onClick={() => handleChangeMode("add")}>+</AddButton>
-            </>
-          </AddItemContainer>
+            </AddItemContainer>
+          ))}
+        {unpurchasedItems.length !== 0 && (
+          <div>
+            <StyledH2>
+              {/* {purchasedItems.length === 0 ? "We need " :  */}
+              There&#39;s {/* } */}
+              {unpurchasedItems.length} thing
+              {unpurchasedItems.length !== 1 ? "s" : ""}{" "}
+              {purchasedItems.length !== 0 ? "left to get" : "on the list"}:
+            </StyledH2>
+            <StyledList>
+              {unpurchasedItems.map((shoppingItem) => {
+                return (
+                  <ShoppingItem
+                    key={shoppingItem.id}
+                    shoppingItem={shoppingItem}
+                    shoppingItemsWithCategoryColor={
+                      shoppingItemsWithCategoryColor
+                    }
+                    backgroundColor={backgroundColor}
+                    onDeleteItem={onDeleteItem}
+                    onEditItem={() => setEditedItem(shoppingItem)}
+                    onChangeMode={() => handleChangeMode("edit")}
+                    onToggleIsPurchased={onToggleIsPurchased}
+                    purchasedItems={purchasedItems}
+                    unpurchasedItems={unpurchasedItems}
+                    shoppingItems={shoppingItems}
+                  />
```

Comment:
your comment above: "You only need to pass the data of the single shopping item to the ShoppingItem component, not the data of all shopping items."
I've taken out the shoppingItems, which works out fine, 
but taking out the purchased and unpurchased items causes an error because I use them in the ShoppingItem component where I conditionally render an input label.

Reply:
Therefore, I will leave them in for now...